src/readNews.js

```diff
@@ -2,6 +2,7 @@ const thenify = require('thenify');
 const open = require('open');
 const invariant = require('invariant');
 const inquirer = require('inquirer');
+const feed = thenify(require('feed-read-parser'));
```

Comment:
This change make the test fail. 
You can fix the test or change the require inside the function. 
(Although I know that require in function is not a proper way but it works.)